app/templates/inbox/meetings/new.hbs

```diff
@@ -0,0 +1,70 @@
+{{!-- TODO: Uncomment this once the app uses `{{page-title}}` on all relevant pages --}}
+{{!-- {{page-title (t "inbox.meetings.new.title")}} --}}
+
+<AuModal
+  @modalTitle={{t "inbox.meetings.new.title"}}
+  @modalOpen={{true}}
+  @closeModal={{this.cancelMeetingCreation}}
+  as |Modal|
+>
+  <Modal.Body>
+    <form class="au-o-flow" id="create-meeting-form" {{on "submit" (perform this.saveMeetingTask)}}>
+      <div>
+        {{#let (unique-id) this.meeting.errors.bestuursorgaan as |id errors|}}
+          <AuLabel
+            @error={{if errors true false}}
+            for={{id}}
+          >
+            {{t "inbox.meetings.new.meeting.administrativeBody"}} <AuPill>Verplicht</AuPill>
+          </AuLabel>
+          <AdministrativeBodySelect
+            @id={{id}}
+            @selected={{this.meeting.bestuursorgaan}}
+            @onChange={{this.updateAdministrativeBody}}
+            @error={{if errors true false}}
+          />
+          {{#each errors as |error|}}
+            <AuHelpText @error={{true}}>
+              {{t error.message}}
+            </AuHelpText>
+          {{/each}}
+        {{/let}}
+      </div>
+      <div>
+        {{#let (unique-id) as |id|}}
+          <AuLabel for={{id}}>{{t "inbox.meetings.new.meeting.location"}}</AuLabel>
+          <AuInput @value={{this.meeting.opLocatie}} @width="block" id={{id}} type="text" />
```

Comment:
I took the liberty of moving this up (so the date inputs are together) and making it full width. I think it looks better :smile:. Let me know if that isn't ok.